src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -1044,7 +1048,37 @@ private void set(String topic, Stack<String> location, CloudioMessageFormat mess
                     CloudioAttribute.InternalAttribute attribute = node.findAttribute(location);
                     if (attribute != null) {
                         // Deserialize the message into the attribute.
-                        messageFormat.deserializeAttribute(data, attribute);
+                        String correlationID = messageFormat.deserializeSetAttribute(data, attribute);
+                        System.out.println(correlationID);
```

Comment:
Remove debug output

Reply:
Oups, someone probably took my computer to add ugly println